install_scripts/bootstrap_arch_linux.sh

```diff
@@ -63,7 +63,7 @@ systemd-machine-id-setup --root="$root_dir" --print
 # The actual Arch Linux setup starts here
 echo_status "Installing base Arch Linux"
 if test -e /etc/arch-release; then
-  pacstrap -c -d "$root_dir" --needed base
+  pacstrap -c -d "$root_dir" --needed base linux linux-firmware
```

Comment:
Note that this script should not have bare-metal-specific config. It is meant to be abstract from the underlying HW/kernel and fit for container setup. Adding those packages here is fine (=container-friendly), although we may also want to eventually refactor the some parts of the scripts to a new "bootstrap_bare_metal.sh".